plugins/image-snapshot/README.md

```diff
@@ -0,0 +1,84 @@
+# @proof-ui/image-snapshot-plugin
+
+Provides Proof with visual regression testing capabilities by porting over functionality from (jest-image-snapshot)[https://github.com/americanexpress/jest-image-snapshot].
+
+## Installation
+
+```bash
+yarn add -D @proof-ui/image-snapshot-plugin
+```
+
+## Usage
+
+```javascript
+// proof.config.js
+import ImageSnapshotPlugin from '@proof-ui/image-snapshot-plugin';
+
+export default {
+  plugins: [
+    new ImageSnapshotPlugin({
+      // Configuration
+    })
+  ]
+};
+```
+
+The image snapshot plugin closely mimics the normal react/jest snapshot testing workflow.  
+
+Start by writing a proof snapshot test for your story. The plugin provides a method on the `browser` object that will create a snapshot for you.
+
+```js
+const proofs = proofsOf('Button');
+
+proofs.add('Basic Usage', async ({ browser }) => {
+  const result = await browser.matchImageSnapshot();
+  expect(result).toBe(true);
+});
+```
+
+Multipe snapshots can be taken per test.
+
+```js
+const proofs = proofsOf('Button');
+
+proofs.add('Basic Usage', async ({ browser }) => {
+  expect(await browser.matchImageSnapshot()).toBe(true);
+  const button = await browser.$('button');
+  await button.click();
+  expect(await browser.matchImageSnapshot()).toBe(true);
```

Comment:
what do think of this expectation footprint. Is there a better way?

Reply:
My suggestion would be to omit the `expect()` and simply let the `await browser.matchImageSnapshot()` throw the exception to fail the test. That would (1) ensure that the error message/stack trace is controlled by the `matchImageSnapshot()` function, and (2) still works if a user forgets to assert the result of the call